src/account/Education.tsx

```diff
@@ -56,96 +58,97 @@ export type Props = NativeSafeAreaViewProps & {
   onFinish: () => void
 }
 
-interface State {
-  step: number
-}
+const Education = (props: Props) => {
+  const {
+    style,
+    embeddedNavBar,
+    stepInfo,
+    buttonType,
```

Comment:
rather than adding `defaultProps` as a static object on this component, you could declare the values here (like `buttonType = BtnTypes.SECONDARY,`) which i think is easier to read / more predictable since you can see it when you're reading the props?